src/sass/_card.scss

```diff
@@ -8,6 +8,28 @@
     padding: 1em;
   }
 
+  .card-image {
+    background-size: 100% 100%;
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
Can we add the following so it's always a nice perspective?

```
  -webkit-background-size: 100%;
  -moz-background-size: 100%;
  -o-background-size: 100%;
  background-size: 100%;
  background-repeat: no-repeat;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  background-position: center center;
  background-attachment: scroll;
```

Reply:
I am curious in why  `background-size` is repeated multiple times

Reply:
oops - you're absolutely right!

should be 

```
  background-repeat: no-repeat;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  background-position: center center;
``

Reply:
Gotcha 💃